index.html

```diff
@@ -1,29 +1,265 @@
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>My Portfolio</title>
-  <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <header class="nav-top">
-    <a class="logo">My Logo</a>
-    <nav class="union"><img src="icons/Union.png"></nav>
-  </header>
-  <section class="headline">
-    <h1 class="text">Hey There. <br> I'm Alex.</h1>
-    <h3 class="text2">I'm a Software Developer</h3>
-    <p class="text3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
-    <ul class="icons">
-    <li><a href="icon"><img src="icons/icon1.png"></a></li>
-    <li><a href="icon"><img src="icons/icon2.png"></a></li>
-    <li><a href="icon"><img src="icons/icon3.png"></a></li>
-    <li><a href="icon"><img src="icons/icon4.png"></a></li>
-    <li><a href="icon"><img src="icons/icon5.png"></a></li>
-    </ul>
-    <div class="illustration"></div>
-    </section>
-</body> 
+    <header class="up-side">
+     <p class="logo">My Logo</p>
+     <nav class="union">
+        <img src="icons/Union.png">
+     </nav>
+     </header>
+     <section class="headline">
+        <h1 class="alex">Hey There. <br> I'm Alex.</h1>
+        <h3 class="soft">I'm a Software Developer</h3>
+        <p class="text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+            <ul class=" icon">
+                <li class="icon1"><img src="icons/icon1.png"></li>
+                <li class="icon2"><img src="icons/icon2.png"></li>
+                <li class="icon3"><img src="icons/icon3.png"></li>
+                <li class="icon4"><img src="icons/icon4.png"></li>
+                <li class="icon5"><img src="icons/icon5.png"></li>
+            </ul>
+            <div class="first-image">
+                <img src="icons/Illustration.png">
+            </div>
+     </section>
+     <section class="work">
+        <h3 class="recent-works">My Recent Works</h3>
+        <div class="vector">
+            <img src="icons/vector.png">
+        </div>
+        <div class="image">
+            <img src="icons/image1.png">
+        </div>
+        <div class="work-text">
+            <h4 class="work-title">Multi Post Stories</h4>
+            <p class="inside-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] Please set up the works section using `grid` and `flexbox` instead of laying the elements in their normal layout, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both)) requirement. This'll make it easier in the future to customize and work on the desktop version.